developer/contributing/pull-requests.rst

```diff
@@ -32,67 +54,116 @@ If you are a member of the `Sulu organization`_ you should also:
 Template
 --------
 
-Template is divided in two parts: *tasks* (for developer) and *information*
-(for reviewer).
+When creating a pull request on GitHub, there is already a pre filled template
+for you. The template is divided in several parts.
 
-The information should include the following fields:
+Checklist
+`````````
+The pull request must include the following checklist at the top to ensure that
+contributions may be reviewed without needless feedback loops and that your
+contributions can be included into Sulu as quickly as possible:
 
-* **Fixed tickets**: **list of tickets**; Tickets to close after merge. These
-  should be of the form ``fixes #123, fixes #321``.
-* **Related PRs**: **link to PRs`**, PRs that are required for the current one
-* **BC breaks**: **none or description**; any compatibility breaking changes
-* **Documentation PR**: **link to PR**; Link to the documentation PR
-  (if applicable)
+.. code-block:: text
 
-.. note::
+    | Q                | A
+    | ---------------- | ---
+    | Bug fix?         | [yes|no]
+    | New feature?     | [yes|no]
+    | BC breaks?       | [yes|no]
+    | Deprecations?    | [yes|no]
+    | Tests pass?      | [yes|no]
+    | Fixed tickets    | [comma separated list of tickets fixed by the PR -> ``fixes #num``]
+    | Related PRs      | [comma separated list of related PRs, if needed]
+    | License          | MIT
+    | Documentation PR | [The reference to the documentation PR if any]
 
-  When an issue number is prefixed with ``fixes`` it tells GitHub to
-  automatically close the referenced ticket when the PR is merged
+An example submission could now look as follows:
+
+.. code-block:: text
 
-You can copy and paste the following template:
+    | Q                | A
+    | ---------------- | ---
+    | Bug fix?         | no
+    | New feature?     | no
+    | BC breaks?       | no
+    | Deprecations?    | no
+    | Tests pass?      | yes
+    | Fixed tickets    | fixes #1243, #1443
+    | Related PRs      | none
+    | License          | MIT
+    | Documentation PR | sulu-io/sulu-docs#153
+
+The whole table must be included (do **not** remove lines that you think are
+not relevant). For simple typos, minor changes in the PHPDocs, or changes in
+translation files, use the shorter version of the check-list:
```

Comment:
Isn't this special rule a bit complicated? I'd stick to one rule for all to keep it simple.